injector.js

```diff
@@ -1,23 +1,26 @@
-var iframe = document.getElementsByTagName('iframe');
+chrome.storage.sync.get(['outputTitle'], ({ outputTitle }) => {
```

Comment:
In manifest v3 api, we cannot send arguments to injector script. One workaround is to use the chrome.storage api to store and retrieve the variables we want.